src/reducers.js

```diff
@@ -1129,6 +1129,17 @@ export const sellItem = (state, { id }, howMany = 1) => {
     : getAdjustedItemValue(valueAdjustments, id)
   const saleIsGarnished = isItemAFarmProduct(item)
   let saleValue = 0
+  let salePriceMultiplier = 1
+
+  if (state.completedAchievements['i-am-rich-3']) {
+    salePriceMultiplier = 1.25
+  } else if (state.completedAchievements['i-am-rich-2']) {
+    salePriceMultiplier = 1.1
+  } else if (state.completedAchievements['i-am-rich-3']) {
+    salePriceMultiplier = 1.05
+  }
```

Comment:
This comment is aimed at your question in the `getItemCurrentValue` util, but makes more sense to answer here.

> should valueAdjustments have a "global" adjustment or something for any permanent global increases?

We should avoid using `valueAdjustments` to calculate sale bonuses, as that object is intended to only define market fluctuations. With that said, my intuition is to move this `salePriceMultiplier` calculation to a helper function in `src/utils.js` and use that helper both here and in `getItemCurrentValue`. What do you think of that approach?

Reply:
i had a feeling `valueAdjustments` was something market specific. can we update the variable name? (is that easy? to like, `marketValueAdjustments`?). or add some documentation around it. i didn't see any explanation for the variable anywhere. whatever you prefer @jeremyckahn . i can also do the work if you let me know which you prefer.

onto the util...that makes sense to me! that will scale nicely later on.

Reply:
lots of unit tests, and a util or two later i think i got there!

Reply:
> i had a feeling `valueAdjustments` was something market specific. can we update the variable name? (is that easy? to like, `marketValueAdjustments`?). or add some documentation around it. i didn't see any explanation for the variable anywhere. whatever you prefer @jeremyckahn . i can also do the work if you let me know which you prefer.

Whoops, I saw this after the fact! Sorry about that. I've effectively addressed this here: https://github.com/jeremyckahn/farmhand/pull/77/files#r641988878

tl;dr: Yes, we should definitely rename `valueAdjustments` to something like `marketValueAdjustments` for the relative value calculations, but we'll also need to having some sort of "grand total" value that is presented to the user as well.